index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+</head>
+
+<body>
+    <nav>
+        <h3 class="logo">My Logo</h3>
+        <img class="onion" src="image/Union.png" alt="" />
+    </nav>
+
+
+    <div class="container">
+
+        <section class="backgroundimg">
+            <div class="menuSection menuSection1">
+                <h2 class="introduction">I’m Michael Glad to see you!</h2>
```

Comment:
- [ ] You should use `h1` tags here instead because of SEO optimization, accessibility by screen readers, essentially proper semantics. If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.
👉 Kindly implement this.